src/panels/media-browser/ha-panel-media-browser.ts

```diff
@@ -0,0 +1,184 @@
+import { mdiLoginVariant } from "@mdi/js";
+import "@polymer/app-layout/app-header/app-header";
+import "@polymer/app-layout/app-toolbar/app-toolbar";
+import {
+  css,
+  CSSResultArray,
+  customElement,
+  html,
+  LitElement,
+  property,
+  TemplateResult,
+} from "lit-element";
+import memoize from "memoize-one";
+import { LocalStorage } from "../../common/decorators/local-storage";
+import { HASSDomEvent } from "../../common/dom/fire_event";
+import { computeDomain } from "../../common/entity/compute_domain";
+import { supportsFeature } from "../../common/entity/supports-feature";
+import "../../components/ha-menu-button";
+import "../../components/media-player/ha-media-player-browse";
+import {
+  BROWSER_SOURCE,
+  MediaPickedEvent,
+  SUPPORT_BROWSE_MEDIA,
+} from "../../data/media-player";
+import "../../layouts/ha-app-layout";
+import { haStyle } from "../../resources/styles";
+import type { HomeAssistant } from "../../types";
+import { showMediaPlayerDialog } from "./show-media-player-dialog";
+import { showSelectMediaSourceDialog } from "./show-select-media-source-dialog";
+
+@customElement("ha-panel-media-browser")
+class PanelMediaBrowser extends LitElement {
+  @property({ attribute: false }) public hass!: HomeAssistant;
+
+  @property({ type: Boolean, reflect: true })
+  public narrow!: boolean;
+
+  // @ts-ignore
+  @LocalStorage("mediaBrowseEntityId")
+  private _entityId?: string;
```

Comment:
Let's default this to the browser
```suggestion
  private _entityId = "browser";
```